src/scss/02_atoms/_buttons.scss

```diff
@@ -554,6 +554,23 @@ $btnsBreakpoints: xsmall small medium large;
   }
 }
 
+.btn--download__icon--svg {
+  align-items: center;
+  color: get($downloadButtonConfig, icon, color);
+  display: flex;
+  font-size: get($downloadButtonConfig, icon, fontSize);
+  height: get($downloadButtonConfig, icon, size);
+  justify-content: center;
+  left: 20px;
```

Comment:
Metti questo valore nella mappa `$downloadButtonConfig` in `rem`.
Oltre questo aggiungi un esempio di utilizzo nel file di test dei bottoni please